### ISSUE TITLE
scale/resize TO 20x20

### ISSUE THREAD

**Issue by nschneid** [carmls/snacs-guidelines#18]
Goal? Extent~>Goal?

[Reddit task_007:526]

**Comment by nschneid** [comment]
cf. change-of-state in #12

**Comment by nschneid** [comment]
If 20x20 is the target size I think it's plain Goal. Consider: "scale it by 50% to 20x20"—50% would be the Extent of the scaling.

**Comment by esmanning** [comment]
Makes sense! In the data it's actually a level of some kind rather than size:

> You can tell Wowhead to scale the search results TO your desired Timewalking level
task_007
new_doc id = 9be3cecb-8c49-7c68-a8c2-4d4244ef7feb
sent_id = 4

but I think the argument still applies
